src/nvim/mouse.c

```diff
@@ -1904,7 +1904,7 @@ static void mouse_check_grid(colnr_T *vcolp, int *flagsp)
 
       // Remember the character under the mouse, might be one of foldclose or
       // foldopen fillchars in the fold column.
-      mouse_char = utf_ptr2char((char *)gp->chars[off]);
+      mouse_char = schar_get_ascii(gp->chars[off]);
```

Comment:
This doesn't look correct. There can be a multibyte char in foldcolumn.

Reply:
right, foldclose/foldopen can be multibyte as well. but looks weird and unreliable to deduce this back from physical grid text, and not the actual state whether this winline had an open or closed fold.